bin/src/report.dart

```diff
@@ -42,9 +42,9 @@ class ReportPart extends CommandLinePart {
 
     if (res.rest.length != 1) return print("Please specify a test file to run");
 
-    var pRoot = new Directory(res["package-root"]);
-    if (!pRoot.existsSync()) return print("Root directory does not exist");
-    log.info(() => "Package root is ${pRoot.absolute.path}");
+    FileSystemEntity pRoot = new File(res["packages"]);
```

Comment:
Should we maybe, as mentioned by @matanlurey, use the package api for this now?
Again for reference:
https://github.com/dart-lang/package_config/blob/master/lib/discovery.dart#L93